### DISCUSSION TITLE
Vite + Storybook project: no such file or directory, open '/vercel/path0/dist/index.d.ts'

### DISCUSSION THREAD

**Discussion by IgnacioNMiranda** [vercel/community#2987] [Help] [answered]
### Page to Investigate

https://github.com/IgnacioNMiranda/vite-component-library-template

### Steps to Reproduce

I've been getting this error for 2 months now:

<img width="1241" alt="image" src="https://github.com/vercel/community/assets/38511917/f1918245-97d0-4aea-9b73-2ff8324ea757">

I have the most updated dependencies, and the build doesn't fail in my local. It seems it was something with the new Storybook 7 version but in [this thread](https://github.com/storybookjs/storybook/issues/22409) it's mentioned that it's not the case anymore, however I'm still experiencing the issue.

Any help would be appreciated.

**Comment by amyegan** [comment, marked as answer]
I was able to get a copy of the repo deployed using the Storybook preset for the project's Build & Development Settings. Does your project use the Storybook preset or a different set of commands?
